game/core/game_view.py

```diff
@@ -199,8 +204,24 @@ def on_update(self, delta_time: float) -> None:
 
     @beartype
     def scroll_to_player(self, speed: float = CAMERA_SPEED) -> None:
+        x = self.player_sprite.center_x
+        y = self.player_sprite.center_y
+
+        if x < 650:
+            x = 650
+        elif x > 3450:
+            x = 3450
+        if y < 300:
+            y = 300
+        elif y > 3700:
+            y = 3700
```

Comment:
Could these be moved to global constants?

Maybe: `MAP_WIDTH = 4_100` and `MAP_HEIGHT = 4_000` then when used with the screen size or `window.width`, the bounds for the camera center positions could be more dynamic

Reply:
Okay I refactored it so the variables are coming from the constants file. I just don't know where 4000 comes from because the map size is 128 x 128 pixels. Maybe I'm missing something obvious? 😅